samples/showcase/src/main/java/com/facebook/fresco/samples/showcase/imageformat/svg/SvgDecoderExample.java

```diff
@@ -57,6 +58,13 @@ public ImageFormat determineFormat(byte[] headerBytes, int headerSize) {
       if (ImageFormatCheckerUtils.startsWithPattern(headerBytes, HEADER)) {
         return SVG_FORMAT;
       }
+      for (String possibleHeaderTag : POSSIBLE_HEADER_TAGS) {
+        byte[] possibleHeader = ImageFormatCheckerUtils.asciiBytes(possibleHeaderTag);
```

Comment:
Doing this for each image is unnecessary work. Since you never actually need the String header tag, you can just do: `POSSIBLE_HEADER_TAGS = { ImageFormatCheckerUtils.asciiBytes("<?xml")}`